keyboards/utd80/keymaps/default/keymap.c

```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
+
+#define ______  KC_TRNS
```

Comment:
This #define is already included in `QMK_KEYBOARD_H` you can delete this

```suggestion

```

Reply:
> This #define is already included in `QMK_KEYBOARD_H` you can delete this

`QMK_KEYBOARD_H` defined seven underscores but I used six. Do I need to change them all?

Reply:
Ideally, yes.  The 7 length one is defined that way on purpose, as we keep the short code keycodes to 7 characters.  That way, they can be consistently aligned in the keymap.